cli/ops/cephadm.py

```diff
@@ -182,3 +182,26 @@ def bootstrap(
         raise OperationFailedError("Failed to bootstrap cluster")
 
     return True
+
+
+def is_mon_stack_enabled(installer, mon_stack):
```

Comment:
instead general method, create methods specific to mon stacks like `is_prometheus_enabled`